### PR TITLE
Add default permissions for page content in a microsites type group. #17

### PR DESCRIPTION
**Pull request by ekes**
So this just adds the default permissions for roles in the group for localgov_page so it rolls out working as expected for now.

Follow-up #20 if we aren't going to have localgov_page always installed. That said it doesn't seem to create a dependency nor get upset if there is a permission for something not there; but that's probably not ideal.

### CODE REVIEW

File: config/install/group.role.microsite-admin.yml
Comment (stephen-cox) [inline config/install/group.role.microsite-admin.yml]: Some UUIDs to remove

File: config/install/group.role.microsite-admin.yml
Comment (stephen-cox) [inline config/install/group.role.microsite-admin.yml]: Needs removing

File: config/install/group.role.microsite-anonymous.yml
Comment (stephen-cox) [inline config/install/group.role.microsite-anonymous.yml]: Remove

File: config/install/group.role.microsite-anonymous.yml
Comment (stephen-cox) [inline config/install/group.role.microsite-anonymous.yml]: Remove

File: config/install/group.role.microsite-member.yml
Comment (stephen-cox) [inline config/install/group.role.microsite-member.yml]: Remove

File: config/install/group.role.microsite-member.yml
Comment (stephen-cox) [inline config/install/group.role.microsite-member.yml]: Remove

File: config/install/group.role.microsite-outsider.yml
Comment (stephen-cox) [inline config/install/group.role.microsite-outsider.yml]: Remove

File: config/install/group.role.microsite-outsider.yml
Comment (stephen-cox) [inline config/install/group.role.microsite-outsider.yml]: Remove

**Review by stephen-cox — APPROVED** [review]
This works for fine page content when accessing them on a custom group domain using the node ID (for example https://localgov-micro-1.lndo.site/node/1) or when it's set as the homepage (for example https://localgov-micro-1.lndo.site/).

Content that has a group alias is not accessible on that alias, but that doesn't look to be a permissions issue.

There are a few UUIDs and hashes in the config that need removing, but approved as long as they;re removed before merging.

**Comment by ekes** [comment]
I'm tempted to write something that greps for `uuid` and `default_config_hash` in config directories before commit!

Removed (having done just that to check which to remove).

**Comment by ekes** [comment]
Squash merged manually.

**Pull request closed without merging**
